src/dms-preview/azext_dms/custom.py

```diff
@@ -169,7 +163,8 @@ def create_task(
         if validate_only is False and validated_task_name is None:
             raise CLIError(
                 "When not validating a MongoDB task, you must supply a previously run 'validate_only' task name.")
-        elif validate_only is False and validated_task_name is not None:
+
+        if validate_only is False and validated_task_name is not None:
```

Comment:
May I know the reason to change from elif to if here?

Reply:
azdev's style checkers prompted this change.